### DISCUSSION TITLE
Filament shield 2.16  - replicate action

### DISCUSSION THREAD

**Discussion by eaziweb95** [filamentphp/filament#4524] [Help] [answered]
I have unselected replicate action in the roles attribution on my entity for one role, but with this role i always see the Replicate link in the table. 
Another remark -->The pushActions in the table seems to be deprecated, is it linked ?

**Comment by danharrin** [comment, marked as answer]
This is a question for the Filament Shield repository, not here

**Reply by eaziweb95** [reply]
The ask question section triggers a 404 error. I report my request in a bug ...
